### PR TITLE
Issue #1163 - Convert CalendarToday to use templates

### PR DESCRIPTION
**Pull request by Mailkov**
-----
IA Page: https://duck.co/ia/view/calendar_today

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: t/CalendarToday.t [hunk 1]
```diff
@@ -89,24 +419,141 @@ ddg_goodie_test(
...
-    "calendar last year" => test_zci(qr/June 2013.*\|11\|/s,
-				     html => qr#<span class="calendar__today circle">11</span>#),
-    "calendar next year" => test_zci(qr/June 2015.*\|11\|/s,
-				     html => qr#<span class="calendar__today circle">11</span>#),
+    "calendar yesterday" => test_zci(
+        qr/June 2014.*\|10\|/s,
+		structured_answer => {
+            id => 'calendar_today',
+            name => 'Answer',
+            data => '-ANY-',
```
Comment (mintsoft) [inline t/CalendarToday.t]: @Mailkov I hate to be awkward, however for `calendar yesterday` and `calendar today` can you assert on the entire `structured_answer` like you have done on `calendar 10 nov 2009`

**Comment by mintsoft** [comment]
@Mailkov Another excellent conversion! Many thanks, I'd just like to make the test coverage around very special queries (`calendar today` for example) a little more explicit then I'm happy to merge :+1:

File: t/CalendarToday.t (hunk 1 above)
Comment (Mailkov) [inline t/CalendarToday.t]: @mintsoft Sure!, there is no problem!

**Comment by moollaza** [comment]
@Mailkov thanks a lot for this!

@mintsoft thanks for reviewing. We should get into the habit of having developers add a link to the IA Page related to the PR so we (currently just me for PR's that update live IAs) can deploy the PR to Beta to really ensure everything works correctly in a production environment.

I'll add the IA Page link here and deploy :+1:

**Comment by mintsoft** [comment]
@moollaza apologies I know you've mentioned that before, it keeps slipping my mind. However I shall endeavour to remember.

**Comment by moollaza** [comment]
@Mailkov Thanks! This LGTM :+1:

I think we may want to port all the logic to JS eventually so that we're not dealing with server time which has the possibility of being different from the users'

**Pull request merged**